### DISCUSSION TITLE
Spacedrive hangs on launch

### DISCUSSION THREAD

**Discussion by lamego** [spacedriveapp/spacedrive#2079] [Community Support]
Installed Spacedrive yesterday, it launched and run without any problems. 
Today it won't run. It hangs on launch, becomes immediately unresponsive.
I'm on an Intel Mac.

Any ideas?

**Comment by utkubakir** [comment]
Interesting. Can you try removing everything here `~/Library/Application\ Support/spacedrive` and trying again?

**Reply by lamego** [reply]
Thanks! Sadly it didn't do anything. Had to delete everything and install 0.2.2 for it to work again.
